### DISCUSSION TITLE
part-ii/2007-2/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#293] [Question comments]
Part II 2007 Number Theory: $3 . \mathrm{I} . 1 \mathrm{~F} \quad$ 

https://questions.tripos.org/part-ii/2007-2/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The continued fraction of $\sqrt{7}$ is $[2,\overline{1,1,1,4}]$. The first few convergents are $\frac{2}{1}$, $\frac{3}{1}$, $\frac{5}{2}$, $\frac{8}{3}$, and we see that $8^2 - 7 \cdot 3^2 = 1$. To find a second solution, we could compute more convergents, but alternatively we can observe that if $x^2 - dy^2 = 1$, then another solution is given by $(x + dy^2, 2xy)$. This gives us the solution $(127,48)$, so two solutions are:

$$(8,3); (127,48)$$

</details>
